aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -614,6 +642,16 @@ module aptos_framework::voting {
         proposal.is_resolved
     }
 
+    #[view]
+    public fun get_resolution_time<ProposalType: store>(
+        voting_forum_address: address,
+        proposal_id: u64,
+    ): u64 acquires VotingForum {
+        let voting_forum = borrow_global<VotingForum<ProposalType>>(voting_forum_address);
```

Comment:
Can you add a separate function for fetching a proposal by id and replace all places where this is done with that function? These 2 lines are repeated in these view functions and elsewhere in the file also so this would be a nice cleanup :)

Reply:
@movekevin done